go.sum

```diff
@@ -0,0 +1,53 @@
+github.com/GehirnInc/crypt v0.0.0-20190301055215-6c0105aabd46/go.mod h1:kC29dT1vFpj7py2OvG1khBdQpo3kInWP+6QipLbdngo=
```

Comment:
Remove go.sum file.
Hint: You can add it to .git/info/exclude so you will not accidentally commit it again.

@emersion, or perhaps we should allow it now?

Reply:
Oops, didn't notice that you removed it in the next commit.

Reply:
I'm fine with allowing it. (In any case not allowing it *and* not putting it in `.gitignore` is silly)

Reply:
Should not be done in this PR though, it's unrelated